core/server/api/db.js

```diff
@@ -51,31 +59,42 @@ db = {
      * @returns {Promise} Success
      */
     importContent: function (options) {
+        var tasks = [];
+
         options = options || {};
 
-        // Check if a file was provided
-        if (!utils.checkFileExists(options, 'importfile')) {
-            return Promise.reject(new errors.NoPermissionError('Please select a file to import.'));
-        }
+        function validate(options) {
+            // Check if a file was provided
+            if (!utils.checkFileExists(options, 'importfile')) {
+                return Promise.reject(new errors.NoPermissionError('Please select a file to import.'));
```

Comment:
@ErisDS I guess those two 2 errors should be ValidationError. If so I'll send PR on it.